engine/tests/phpunit/Elgg/Application/ServeFileHandlerTest.php

```diff
@@ -28,7 +28,11 @@ public function setUp() {
 
 		$file = new \ElggFile();
 		$file->owner_guid = 1;
-		$file->setFilename("foobar.txt");
+
+		// Using special characters to test against files that have been
+		// uploaded prior to implementation of filename sanitization
+		// See #10608
+		$file->setFilename("foo'baž.txt");
```

Comment:
I think filesystem name mapping issues are causing the tests to not find this file, so I'm going to try reading the name from disk.